xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-include/src/main/java/org/xwiki/rendering/macro/include/IncludeMacroParameters.java

```diff
@@ -151,7 +157,30 @@ public String getSection()
     {
         return this.section;
     }
-
+    
+    /**
+     * @param excludeHeading {@code true} to remove the first heading found inside
+     *         the document or a section, {@code false} to keep it 
```

Comment:
if you say "the document" then you need to say "the section" too...  You need consistency.

--> "... found inside the included document or section..."